pandas/_config/__init__.py

```diff
@@ -2,3 +2,5 @@
 pandas._config is considered explicitly upstream of everything else in pandas,
 should have no intra-pandas dependencies.
 """
+__all__ = ["config"]
+from . import config
```

Comment:
can you use absolute imports

Reply:
I guess, but relative import here is more in the spirit of "self-contained"